### PR TITLE
Move to native packer (from assembly), adapt doc & clean

### PR DESCRIPTION
**Pull request by eloots**
- Using sbt Native Packager (instead of sbt)
    - Faster.
    - No config merging issues.
- Adapter `copy` and `run` scripts to take into account move to native
  packager
- Update CMT Man plugin code.
- Remove unneeded exercise README.md file in root project (possible as
  the CMT config was changed to look for README files in a project's
  root folder.
- Clean-up/re-format `project/CommonSettings.scala` & add more
- Remove sbt assembly plugin from configuration
- Adapt Akka debug level & logback config

### CODE REVIEW

**Comment by eloots** [comment]
See #123 -  to be fixed soon

File: docs/Hypriot-OS-Course-Preparation-Instructions.md [hunk 1]
```diff
@@ -21,21 +21,17 @@ On your Mac, add the following entries to your _/etc/hosts_ file:
 
 Next, we set-up password-less login (unless your `ssh` remembers your password, in which case you can skip this step). We assume that we want to log in to account _akkapi_ on a Raspberry Pi node from an account (_userxxx_) on your laptop. In order to set-up password-less login, you need a so-called public/private key pair. If you're unfamiliar with this concept, or don't have such a key pair, first read the paragraph titled _Generating a public/private key pair_ in the Addendum.
```
Comment (kikiya) [inline docs/Hypriot-OS-Course-Preparation-Instructions.md]: one thing to note, this does not work out on Windows machines sometimes.

File: exercise_013_clustered_sudoku_solver/src/main/scala/org/neopixel/SudokuIO.scala [hunk 2]
```diff
@@ -54,20 +56,43 @@ object SudokuIO {
...
   import scala.language.postfixOps
-  class FileLineTraversable(file: File) extends Traversable[String] {
-    override def foreach[U](f: String => U): Unit = {
-      val input = new BufferedReader(new FileReader(file))
-      try {
-        var line = input.readLine
-        while (line != null) {
-          f(line)
-          line = input.readLine
+  class FileLineTraversable(file: File) extends Iterable[String] {
```
Comment (kikiya) [inline exercise_013_clustered_sudoku_solver/src/main/scala/org/neopixel/SudokuIO.scala]: why does this have to be traversable? or why do we need to write our own traversable logic?

**Comment by eloots** [comment]
Is that so even with the WSL (Windows Subsystem for Linux) on Windows 10?

The Wikipedia page on WSL (https://en.m.wikipedia.org/wiki/Windows_Subsystem_for_Linux) gives more details, but it’s unclear if it’s installed on an average windows 10 installation...

Verstuurd vanaf mijn iPhone

**Comment by kikiya** [comment]
@eloots I noticed that Windows problem during the Scala days workshop in Lausanne. That student was using the subsystem but I'm not entirely sure it was the most updated version.

**Comment by eloots** [comment]
> @eloots I noticed that Windows problem during the Scala days workshop in Lausanne. That student was using the subsystem but I'm not entirely sure it was the most updated version.

@kikiya Indeed, I remember that... except for the fine details. I guess that one's mileage may vary, depending on the version/edition of Windows 10. I think that anything older than Windows 10 is a no-go...

**Comment by eloots** [comment]
@kikiya @agolubev I added some more checking & coloured output + instructions about how to correct the node installation in case the `unzip` command is missing.

**Comment by eloots** [comment]
Force pushed after rebase on master

**Pull request merged**